pallets/gear/src/lib.rs

```diff
@@ -73,6 +69,12 @@ use sp_std::{
     prelude::*,
 };
 
+#[cfg(feature = "std")]
+type ExecutionEnvironment = gear_backend_wasmi::WasmiEnvironment<Ext>;
+
+#[cfg(not(feature = "std"))]
+type ExecutionEnvironment = gear_backend_sandbox::SandboxEnvironment<Ext>;
```

Comment:
Don't you want to define Ext in the same manner dependent on `lazy-pages` feature to keep everything variable close?